src/MizarWidgets/SamplingWithFrameTrackWidget.ui

```diff
@@ -34,6 +34,9 @@
          <height>30</height>
         </size>
        </property>
+       <property name="toolTip">
+        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;False alarm &lt;/span&gt;is an event of reporting statistically significant difference in the absence of such. In our case, an event of the tool reporting that the runtime of a function significantly differs between the captures by mistake, not due to the presence of real difference. &lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;When the multiplicity correction is &lt;span style=&quot; font-style:italic;&quot;&gt;off, &lt;/span&gt;we control the probability of false alarm &lt;span style=&quot; font-style:italic;&quot;&gt;for each function&lt;/span&gt; is controlled. &lt;/p&gt;&lt;p&gt;When the multiplicity correction is &lt;span style=&quot; font-style:italic;&quot;&gt;on&lt;/span&gt;, the probability of at least one false alarm is controlled. &lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;The probability can be chosen in the drop down list to the right.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
It's hard to read, but I think it says "When the multiplicity correction is _off_, we control the probability of false alarm for each function is controlled." - if I parsed it correctly, then "is controlled" should be removed as it's already the mentioned in active voice.  As for when it's off, simply saying it also says that it "is controlled" but that sounds like it's controlled by the program, if it's controlled by the user, I would say "the probability of at least one false alarm is controlled (by you | by the drop down list to the right)."

I would also add "A" before each case of "false alarm".

Sorry if I'm misunderstanding the meaning :S

Reply:
Sorry, my bad. So now it's 

```
A false alarm is an event of reporting statistically significant difference in the absence of such. In our case, an event of the tool reporting that the runtime of a function significantly differs between the captures by mistake, not due to the presence of real difference. 
When the multiplicity correction is off, the probability of a false alarm for each function is controlled by the drop down list to the right. 
When the multiplicity correction is on, the probability of at least one false alarm is controlled by the drop down list to the right. 
```

Reply:
... or rather ` by the same drop down list` at the end.

Reply:
Ah okay, I also misunderstood it before; that looks good now. Another way of phrasing it if you're trying to avoid repeating the drop down list could be something like:

The probability of a false alarm is controlled by the drop down list to the right.  When the multiplicity correction is off, the probability applies to each function. When the multiplicity correction is on, it's the probability of at least one false alarm in all of the functions. (or the whole capture? not sure exactly what this should reference)

But I'm fine with either.